robot/EDA/resources/EDA.py

```diff
@@ -418,3 +418,33 @@ def verify_dropdown_field_status(self, **kwargs):
             expected_value = bool(expected_value == "disabled")
             if not str(expected_value).lower() == str(actual_value).lower() :
                 raise Exception (f"Drop down field {field} status is {actual_value} instead of {expected_value}")
+
+    def verify_action_button_status(self, **kwargs):
+        """ Verify the action button is disabled/enabled for the user
+            we have to pass the name of the button and the expected status
+            of the action button as either enabled or disabled
+        """
+        for button,expected_value in kwargs.items():
+            locator = eda_lex_locators["eda_settings"]["run_action"].format(button)
```

Comment:
Since this keyword is in EDA.py, maybe it makes sense to rename the locator to "action_button" instead of "run_action" ??